tools/pckgen.d/rpm/CAS_NAME.spec

```diff
@@ -86,15 +86,21 @@ fi
 %post modules_%{kver_filename}
 depmod
 . /etc/os-release
+# Determine the exact location of installed modules to add them to weak-modules
+for file in $(rpm -ql $(rpm -qa | grep <CAS_NAME>-modules)); do
+if [[ "$file" =~ cas_.*\.ko$ ]]; then
+    # realpath to resolve any possible symlinks (needed for weak-modules)
+    modules+=( $(realpath "$file") )
+fi
+done
```

Comment:
I'm not sure if this will list all CAS modules installed in all kernel directories or just modules installed in current kernel directory (because 'rpm -ql' lists files that belong to the package and not any possible links created by module linking mechanism). But I believe Jan was testing this and it worked as it supposed to.
(also the indent in 'if-fi' is gone)